examples/common_vmseries_and_autoscale/example.tfvars

```diff
@@ -0,0 +1,247 @@
+# --- GENERAL --- #
+location            = "North Europe"
+resource_group_name = "autoscale-common"
+name_prefix         = "example-"
+tags = {
+  "CreatedBy"   = "Palo Alto Networks"
+  "CreatedWith" = "Terraform"
+}
+enable_zones = false
```

Comment:
dedicated vmss is non-zonal. Maybe we could make this example zone-enabled?

Reply:
converted the example to zone-enabled but there is a couple of things to note; since NATGWs are zonal resources it's removed from the example and the outbound internet traffic is provided from public IP addresses assigned to public interfaces of firewalls in the VMSS. 
Another point is since we don't have a NATGW, I needed to enable public IPs on management interfaces as well, otherwise it wouldn't be possible for the firewalls to bootstrap with publicly accessible Panorama or do their initial licensing with Palo Alto update servers since they won't have internet access. 
We might add a note for this on README perhaps; mentioning that public IP addresses on management interfaces can be removed after the deployment and all management traffic can be forwarded to the firewalls' private interfaces to get NATed to the internet (after the firewalls get configured to NAT this traffic). Any other ideas?

Reply:
> Another point is since we don't have a NATGW, I needed to enable public IPs on management interfaces as well, otherwise it wouldn't be possible for the firewalls to bootstrap with publicly accessible Panorama or do their initial licensing with Palo Alto update servers since they won't have internet access.

that's not entirely true. A VM with just private IP and default NSG rules (or no NSG) can access the internet, it's not accessible from the internet. So you don't need public IPs on management to contact a publicly available Panorama (which probably will never happen) or to download updates.
In our examples we do not block the default outbound route, so we do not need public IPs for that.

Reply:
After testing and troubleshooting with public IPs removed from management interfaces, we have found out that default outbound access is blocked if VMs are attached to a Load Balancer or they have any additional interface with a public IP assigned. We have both so relying on the default outbound access won't be possible. 

And since NATGWs are zonal, this leaves us with 2 feasible options I believe:
* Assigning public IP to both public and management interfaces
* Creating a separate backend pool for the management interfaces in order to create outbound rules from them (but this is problematic as well since we don’t support creating separate backend pools for different interfaces on our current LB module)

As a note if we assign public IPs to the management interfaces and the user wants to remove them we can put a comment on README saying that public IP addresses on management interfaces can be removed after the deployment with management traffic forwarded to the firewalls’ private interfaces (private LB) to get NATed to the internet (after the firewalls get configured to NAT this traffic).

Reply:
It is agreed to keep public IP addresses on management interfaces for the simplicity of the example and put a disclaimer on how to avoid it.